portfolio/src/main/webapp/script.js

```diff
@@ -31,9 +33,10 @@ function showProjectDetail(buttonId) {
       ];
 
   // Add description to the page. 
-  const descriptionDiv = document.getElementById(buttonId+"-div");
-
-  switch(buttonId) {
+  const descriptionDiv = document.createElement('div');
+  const targetElement = event.target;
+  
+  switch(targetElement.id) {
     case "a":
```

Comment:
Is there a better way to map a, b, c, d etc. to the array values? Can you use numbers instead of letters?

Reply:
Done.